_data/community.yml

```diff
@@ -195,6 +215,14 @@
             date: 2023-09-22
             link: https://socrates-conference.at/
             cost: 39 EUR
+-   id: tedxlinz
+    name: "TEDxLinz"
+    events:
+        -   name: "TEDxLinz To Boldly Go"
+            location: LIT, JKU
+            date: 2023-11-18
+            link: https://www.tedxlinz.at/
+            cost: 79 EUR
```

Comment:
Please check our FAQs concerning adding paid events, before we merge: https://technologieplauscherl.at/faq/

Reply:
while it is a paid event, it's not for profit or even paying wages to anyone, just to cover costs.
If this is not enough i'll remove it from the MR, there are no free tickets in the budget :)

Reply:
Thanks for the clarification